R/dataframe.R

```diff
@@ -76,7 +76,7 @@ lst_ <- function(xs) {
     deparse2 <- function(x) paste(deparse(x$expr, 500L), collapse = "")
     defaults <- vapply(xs[missing_names], deparse2, character(1),
       USE.NAMES = FALSE)
-
+    defaults <- ifelse(defaults == "NULL", "", defaults)
```

Comment:
Thanks. I see your intention, but I prefer the behavior to be consistent with passing other unnamed arguments.